config/CHANGELOG.md

```diff
@@ -11,6 +11,82 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 [comment]: <> (Fixed:      any bug fixes)
 [comment]: <> (Security:   in case of vulnerabilities)
 
+## 2.0.0 - 17889
+### casper-node 2.0.0-a7e4ff1
+
+## This was a massive breaking change update and the change log is calling out major pieces but not exhaustive.
+
+## Changed
+* config-example.toml
+  * network
+    * known_addresses
+    * blocklist_retain_duration
+      * Changed to min and max times to allow a more randomized timing of block list between these values.
```

Comment:
* Changed to min and max times. Each time a peer is blocked a value is randomly chosen between these values. This smooths recovery from network partitioning and various other edge cases.